news_items/004_freq_testing_paper/index.html

```diff
@@ -0,0 +1,54 @@
+</p>
+
+<div class="d-flex justify-content-center">
+  <div class="card border-1 mt-2 mb-4" style="width: 90%;">
+    <div class="text-center m-4">
+      <a href="exam_schedule.png"><img src="exam_schedule.png" class="card-img-top" alt="Traditional versus frequent exam schedules."></a>
+    </div>
+    <div class="card-body d-flex justify-content-center bg-light">
+      <p class="card-text">
+        <b>Figure 1:</b> Traditional versus frequent exam schedule in TAM 251.
+      </p>
+    </div>
+  </div>
+</div>
+
+<p>
+  <b>How much did student learning improve?</b> An identical pen-and-paper final exam was given in both semesters. Students using the frequent exam schedule scored 7% higher on average (about 2/3 of a letter grade), despite the fact that they had not practiced a pen-and-paper exam before in this class. This resulted in twice the number of A grades on the final exam and a drop of 60% in D and F grades (see Figure 2). There were over 200 students in each of the traditional- and frequent-exam semesters so all the results are statistically significant. As shown in the paper, women and minority students had the same increases as the rest of the population.
+</p>
+
+<div class="d-flex justify-content-center">
+  <div class="card border-1 mt-2 mb-4" style="width: 80%;">
+    <div class="text-center m-4">
+      <a href="exam_score_dist.png"><img src="exam_score_dist.png" class="card-img-top" style="max-width: 40rem;" alt="Histogram of exam scores."></a>
+    </div>
+    <div class="card-body d-flex justify-content-center bg-light">
+      <p class="card-text">
+        <b>Figure 2:</b> Histogram of student results on the identical pen-and-paper final exam for the two semesters. Exam letter grades correspond to the following exam score ranges: A (90% or higher), B (80% to 90%), C (70% to 80%), and D/F (below 70%). Error bars show 95% confidence intervals.
+      </p>
+    </div>
+  </div>
+</div>
+
+<p>
+  <b>What's the theory behind these results?</b> Overall, it seems likely that students both study harder and use more effective study strategies when given frequent tests, for at least three reasons. First, having many small exams essentially forces students to use <a href="https://en.wikipedia.org/wiki/Distributed_practice">distributed practice</a> rather than cramming. Second, testing is simply more effective at solidifying memory than just studying (the <a href="https://en.wikipedia.org/wiki/Testing_effect">testing effect</a> in psychology). Third, second-chance exams and mastery grading encourage students to learn from their errors on the first-chance exams, rather than simply dismissing the results as a fluke or the result of an unfair test, thereby improving <a href="https://en.wikipedia.org/wiki/Metacognition">metacognition</a>.
```

Comment:
Any data to justify studying harder? (Self-reported metrics? Or PL session times on practice/hw?) Could it be similar problem types appeared on both the assessment and the final? So, more of a transference?

Reply:
I changed to "study more" -- this is just a comment that they had a lot more study sessions (up to 14 exams rather than 2). The paper did not do a detailed analysis of question similarity, so I won't speculate about that.